xwiki-platform-core/xwiki-platform-ckeditor/xwiki-platform-ckeditor-test/xwiki-platform-ckeditor-test-docker/src/test/it/org/xwiki/ckeditor/test/ui/ImagePluginIT.java

```diff
@@ -549,17 +573,85 @@ void updateExternalImageSize(TestUtils setup, TestReference testReference) throw
 
         assertEquals("[[image:" + imageURL + "||height=\"50\" width=\"50\"]]", savedPage.editWiki().getContent());
     }
+    
+    @Test
+    @Order(13)
+    void quickInsertImageThisPage(TestUtils setup,
+            TestReference testReference) throws Exception
+    {
+        
+        // Upload an attachment to the page.
+        String attachmentName = "image.gif";
+        AttachmentReference attachmentReference = new AttachmentReference(attachmentName, testReference);
+        ViewPage newPage = uploadAttachment(setup, testReference, attachmentName);
+        
+        // Move to the WYSIWYG edition page.
+        WYSIWYGEditPage editPage = newPage.editWYSIWYG();
+        this.editor = new CKEditor("content").waitToLoad();
+        this.textArea = this.editor.getRichTextArea();
```

Comment:
We're missing:

```
void edit(TestUtils setup, TestReference testReference, boolean startFresh)
```

in ``AbstractCKEditorIT`` so that you can write:

```suggestion
        edit(setup, testReference, false);
```

Reply:
Thanks, I will add the method